pyne/mesh.py

```diff
@@ -704,6 +704,18 @@ def __init__(self, mesh=None, structured=False,
             stored in self.dims.
 
         """
+	# if Mesh is made and no parameters passed, raise MeshError
+	if mesh is None and not structured and structured_coords is None and \
+	structured_set is None and structured_ordering=='xyz' and \
+	mats==():
```

Comment:
```suggestion
	    mats==():
```

Reply:
maybe consider moving the `mats==():` line to the previous one if it fits (<80 char/line)